hostmetrics/hostmetrics-with-variable.json

```diff
@@ -227,17 +227,17 @@
       "y": 16
     }
   ],
+  "name": "",
   "tags": ["hostmetrics", "signoz"],
   "title": "HostMetrics Dashboard - SigNoz",
   "variables": {
     "hostname": {
-      "allSelected": false,
       "customValue": "",
       "description": "Hostname of the instance",
-      "modificationUUID": "d7bda50a-b877-493d-99e4-df1f365dbbb2",
       "multiSelect": false,
+      "name": "hostname",
       "queryValue": "SELECT DISTINCT(JSONExtractString(labels, 'host_name'))\nFROM signoz_metrics.time_series_v2",
```

Comment:
you wanted to update the query for non-k8s environments?

Reply:
yeah, I had missed out on pushing that commit. it should be there now.